pytest-selenium/pages/content.py

```diff
@@ -0,0 +1,45 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+
+from pages.base import Page
+from regions.base import Region
+
+
+class Content(Page):
+    URL_TEMPLATE = "/books/{book_slug}/pages/{page_slug}"
+
+    _main_content_locator = (
+        By.CSS_SELECTOR,
+        '#main-content > div > div > div h1[data-type="document-title"]',
+    )
+    _table_of_contents_button_locator = (By.CSS_SELECTOR, "#root > div > div > button")
+    _table_of_contents_nav_locator = (By.CSS_SELECTOR, "#root > div > div > div > div > nav")
```

Comment:
```suggestion
    _table_of_contents_nav_locator = (By.CSS_SELECTOR, "nav")
```

Reply:
Did you test this change?

Reply:
eep, there are 36 `<nav>` elements. I think that's a bug... there should only be one.

Reply:
This will probably be fixed to be just one `<nav>` as part of the "Add ToC Toggle" story in "Reading Experience": https://docs.google.com/document/d/1-KX5NMSeJTP5309L7mlXw85YHG_7XRFm-C7ofIdWtCE/edit#heading=h.apfasqkrfa3g

Reply:
@philschatz the nav element is used for navs and subnavs, so each chapter has its own nav. i don't know of any reason to limit to just 1 nav per page

Reply:
From the [MDN Description of nav](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) it seems like they are used to denote separate navigations on a page (e.g. ToC, site navigation) and are not nested. In the epub spec the entire ToC is a `<nav>` which contains nested `<ul>`, `<li>`, and `<a href="">` elements.

Reply:
ok, we can definitely change it to only be one nav for the ToC, we may end up having another one for site nav though so we should assume there are at least a couple

Reply:
Yeppers, and when a 2nd `<nav>` is introduced they can be differentiated via `aria-label` attributes so the selenium tests _and_ screenreaders can differentiate 😄